workspace/ai/tests/test_llm.py

```
@@ -0,0 +1,94 @@
+    def test_openai_like_form_with_dict_arguments(self):
+        content = '{"name": "create_event", "arguments": {"title": "Lunch", "duration": 30}}'
+        calls, remaining = extract_text_tool_calls(content)
+        self.assertEqual(len(calls), 1)
+        name, args_json = calls[0]
+        self.assertEqual(name, 'create_event')
+        self.assertEqual(json.loads(args_json), {'title': 'Lunch', 'duration': 30})
+        self.assertEqual(remaining, '')
+
+    def test_openai_like_form_with_string_arguments(self):
+        content = '{"name": "echo", "arguments": "raw string"}'
+        calls, remaining = extract_text_tool_calls(content)
```

Ruff (0.15.12)

[warning] Unpacked variable `remaining` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)